hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/TestHoodieTimelineArchiveLog.java

```diff
@@ -495,11 +496,16 @@ private void createReplaceMetadata(String instantTime) throws Exception {
     String fileId2 = "file-" + instantTime + "-2";
 
     // create replace instant to mark fileId1 as deleted
+    HoodieRequestedReplaceMetadata requestedReplaceMetadata = HoodieRequestedReplaceMetadata.newBuilder()
+        .setOperationType(WriteOperationType.INSERT_OVERWRITE.toString())
+        .setVersion(1)
```

Comment:
Should this be the CURRENT_VERSION from timeline layout ?

Reply:
@n3nash This is not related to timeline layout version. This is tracking version of HoodieRequestedReplaceMetadata (in case we add new fields to replace metadata avro structure, we can increase version). 

In the test, we were actually already configured to use default timeline layout version 1. But test was only creating commit file and was not creating requested files. I modified to add requested files as it happens with timeline layout version 1.